distribution/lib/Standard/Base/0.0.0-dev/src/Network/HTTP/Header.enso

```diff
@@ -89,14 +91,13 @@ type Header
              example_auth_basic = Header.authorization_basic "user" "pass"
     authorization_basic : Text -> Text -> Header
     authorization_basic user pass =
```

Comment:
```suggestion
    authorization_basic : Text -> Text|Enso_Secret -> Header
    authorization_basic (user : Text) (pass : Text | Enso_Secret) =
```

Reply:
Ah ok I see that this would complicate the logic quite significantly.

I think useful to allow this (together with also allowing secrets in `authorization_bearer`) at some point, but probably not at this PR.